src/components/Filters.js

```diff
@@ -57,25 +81,93 @@ function Filters() {
               <span>Sort By</span>
             </div>
             <label>
-              <input type="radio" name="sort" />
+              <input
+                type="radio"
+                name="sort"
+                onChange={(e) => {
+                  filterDispatch({
+                    type: "LOW_TO_HIGH",
```

Comment:
you can give the type name as sort by to indicate that it's for sorting as you have done for categories to indicate categories also both type and payload are named same, so can change the name.